cime/config/e3sm/machines/config_pio.xml

```diff
@@ -123,13 +123,13 @@
     </values>
   </entry>
   -->
-  <entry id="LND_PIO_REARRANGER">
+<!--  <entry id="LND_PIO_REARRANGER">
```

Comment:
How about removing these entries instead of leaving them as comments?

Reply:
It might be useful to retain this entry to show how to use it in future.

Reply:
Ok